Lib/ufo2ft/outlineOTF.py

```diff
@@ -375,49 +375,59 @@ def setupTable_OS2(self):
         os2.usWidthClass = getAttrWithFallback(font.info, "openTypeOS2WidthClass")
         # embedding
         os2.fsType = intListToNum(getAttrWithFallback(font.info, "openTypeOS2Type"), 0, 16)
-        # subscript
+
+        # subscript, superscript, strikeout values, taken from AFDKO:
+        # FDK/Tools/Programs/makeotf/makeotf_lib/source/hotconv/hot.c
+        unitsPerEm = getAttrWithFallback(font.info, "unitsPerEm")
+        italicAngle = getAttrWithFallback(font.info, "italicAngle")
+        xHeight = getAttrWithFallback(font.info, "xHeight")
+        def adjustOffset(offset, angle):
+            """Adjust Y offset based on italic angle, to get X offset."""
+            return offset * math.tan(math.radians(-angle)) if angle else 0
+
         v = getAttrWithFallback(font.info, "openTypeOS2SubscriptXSize")
         if v is None:
-            v = 0
+            v = unitsPerEm * 0.65
```

Comment:
I thought that the reference to FDK/Tools/Programs/makeotf/makeotf_lib/source/hotconv/hot.c was enough (it may be the best I can do, since I don't know where that code gets these values). I could include the entire hyperlink as I did in the commit message if preferred.